### PR TITLE
Making sure referenced repository resources exist

### PR DESCRIPTION
**Pull request by escowles**
Fixes https://jira.duraspace.org/browse/FCREPO-2234

### CODE REVIEW

File: src/main/java/org/fcrepo/importexport/importer/Importer.java [hunk 1]
```diff
@@ -219,11 +219,32 @@ private Model sanitize(final Model model) {
...
 
+    /**
+     * Make sure that a URI exists in the repository.
+     */
+    private void ensureExists(final URI uri) throws FcrepoOperationFailedException {
+        final String rdf = "<> a <http://www.w3.org/ns/ldp#Container> .";
+        final FcrepoResponse response = client().put(uri)
+                .body(new ByteArrayInputStream(rdf.getBytes()), "text/turtle")
+                .preferLenient().perform();
+        if (response.getStatusCode() > 409) {
```
Comment (mikedurbin) [inline src/main/java/org/fcrepo/importexport/importer/Importer.java]: We could fail with 3xx errors too.

File: src/test/java/org/fcrepo/importexport/integration/ImporterIT.java [hunk 2]
```diff
@@ -107,6 +107,33 @@ public void testImport() throws FcrepoOperationFailedException, IOException {
...
+        config.setPassword(PASSWORD);
+
+        // run import
+        final Importer importer = new Importer(config, clientBuilder);
+        importer.run();
+
+        // verify the resources exist
+        resourceExists(linkFrom);
+        resourceExists(linkTo);
+    }
```
Comment (mikedurbin) [inline src/test/java/org/fcrepo/importexport/integration/ImporterIT.java]: Could we go the extra mile and ensure that the links were actually created as well?

**Comment by awoods** [comment]
Looks good to me, pending response to @mikedurbin's comments.

File: src/main/java/org/fcrepo/importexport/importer/Importer.java [hunk 3]
```diff
@@ -219,11 +219,32 @@ private Model sanitize(final Model model) {
...
         }
         return model.remove(remove);
     }
 
+    /**
+     * Make sure that a URI exists in the repository.
+     */
+    private void ensureExists(final URI uri) throws FcrepoOperationFailedException {
+        final String rdf = "<> a <http://www.w3.org/ns/ldp#Container> .";
+        final FcrepoResponse response = client().put(uri)
```
Comment (escowles) [inline src/main/java/org/fcrepo/importexport/importer/Importer.java]: That's a good idea — I'll add that too.

**Comment by ajs6f** [comment]
@ruebot Give @escowles  a sec for [this](https://github.com/fcrepo4-labs/fcrepo-import-export/pull/40#discussion_r79891204).

**Pull request merged**